### PR TITLE
zh-TW: Translate testing.md and update CODEOWNERS

### PR DESCRIPTION
**Pull request by johnathan79717**
I know that "supported" is directly translated to "支援", but I somehow feel awkward to use that in a sentence.  I welcome ideas and suggestions.

### CODE REVIEW

File: po/zh-TW.po [hunk 1]
```diff
@@ -8937,18 +8937,21 @@ msgstr ""
 
 #: src/testing.md:1
 msgid "# Testing"
-msgstr ""
+msgstr "# 測試"
 
 #: src/testing.md:3
 msgid "Rust and Cargo come with a simple unit test framework:"
-msgstr ""
+msgstr "Rust 和 Cargo 帶有一個簡單的單元測試 (unit test) 框架："
```
Comment (victorhsieh) [inline po/zh-TW.po:8944]: 帶有 -> 提供了?
：-> ": " (半型加空白)

File: po/zh-TW.po [hunk 2]
```diff
@@ -8937,18 +8937,21 @@ msgstr ""
...
 
 #: src/testing.md:5
 msgid ""
 "* Unit tests are supported throughout your code.\n"
 "\n"
 "* Integration tests are supported via the `tests/` directory."
 msgstr ""
+"* 單元測試在你的程式碼的任何地方都可添加。\n"
```
Comment (victorhsieh) [inline po/zh-TW.po:8952]: 單元測試在你的程式碼的任何地方都可添加 -> 在你的程式碼的任何地方都可添加單元測試?

File: po/zh-TW.po [hunk 3]
```diff
@@ -8937,18 +8937,21 @@ msgstr ""
...
+"\n"
+"* 集成測試 (integration test) 則在 `tests/` 資料夾下可添加。"
```
Comment (victorhsieh) [inline po/zh-TW.po:8954]: 則在 `tests/` 資料夾下可添加 -> 則可以放置在 `tests/` 資料夾下

**Review by victorhsieh — CHANGES_REQUESTED** [review]

File: po/zh-TW.po (hunk 3 above)
Comment (mingyc) [inline po/zh-TW.po:8954]: > 集成測試 (integration test)

I feel that `集成測試` is not a native term in zh-TW (correct me if I am wrong).

According to [wiki TW](https://zh.wikipedia.org/wiki/%E9%9B%86%E6%88%90%E6%B5%8B%E8%AF%95), it should be `整合測試`.

**Comment by johnathan79717** [comment]
Thanks for the suggestions.  I updated the commit!

**Review by victorhsieh — APPROVED** [review]

**Pull request merged**